aten/src/ATen/Declarations.cwrap

```diff
@@ -69,6 +69,8 @@
 [[
   name: _th_masked_fill_
   cname: maskedFill
+  cpu_bool: True
```

Comment:
This allows you to have the value/return be a bool tensor, which I guess is fine, but doesn't really have anything to do with the other part of this PR, which is putting a deprecation warning on byte masks.  To enable a deprecation path you'd need to support the mask being a bool.